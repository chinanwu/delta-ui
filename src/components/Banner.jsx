import React from 'react';

import './Banner.less';

export const Banner = () => (
	<div className="Banner">
		<div className="Banner__message">
			<a className="Banner__link" href="https://linktr.ee/blacklivesmatter">
				Black Lives Matter
			</a>
		</div>
	</div>
);

export default Banner;
