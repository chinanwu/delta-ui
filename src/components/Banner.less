.Banner {
	background-color: black;
	display: flex;
	justify-content: center;
}

.Banner__message {
	display: flex;
	align-items: center;
	height: 4rem;
	font-size: 2rem;
	color: white;
}

.Banner__link {
	cursor: pointer;

	&:link {
		color: white;
	}

	&:visited {
		color: white;
	}
}
