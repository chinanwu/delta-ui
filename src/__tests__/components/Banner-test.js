import { shallow } from 'enzyme';
import React from 'react';

import { Banner } from '../../components/Banner.jsx';

jest.unmock('../../components/Banner.jsx');

describe('Banner component', () => {
	describe('rendering', () => {
		it('renders Banner', () => {
			const wrapper = shallow(<Banner />);
			expect(wrapper.getElement()).toMatchSnapshot();
		});
	});
});
