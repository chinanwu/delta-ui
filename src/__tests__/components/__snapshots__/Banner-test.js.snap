// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner component rendering renders Banner 1`] = `
<div
  className="Banner"
>
  <div
    className="Banner__message"
  >
    <a
      className="Banner__link"
      href="https://linktr.ee/blacklivesmatter"
    >
      Black Lives Matter
    </a>
  </div>
</div>
`;
